pai-management/paiLibrary/common/file_handler.py

```diff
@@ -69,3 +69,39 @@ def file_delete(file_path):
 def create_folder_if_not_exist(folder_path):
     if not os.path.exists(folder_path):
         os.makedirs(folder_path)
+
+
+def add_deploy_rule_to_yaml(service_conf, src_yaml, dst_yaml):
+    
```

Comment:
Suggest to add some lines of comments to explain the generate affinity YAML context for in and not in so that user can understand this section of code easily.

Reply:
OK, Added